tstest/clock.go

```diff
@@ -303,6 +303,12 @@ func (c *Clock) AfterFunc(d time.Duration, f func()) tstime.TimerController {
 	return t
 }
 
+// Since subtracts specified duration from Now().
+func (c *Clock) Since(t time.Time) time.Duration {
+	c.init()
```

Comment:
Hmm. `c.Now()` will init so it's probably not necessary, but I guess leaving it here maintains the pattern.